unique_toolkit/unique_toolkit/content/functions.py

```diff
@@ -103,4 +188,272 @@ def upsert_content(
         scopeId=scope_id,
         chatId=chat_id,
         fileUrl=file_url,
-    )
+    )  # type: ignore
+
+
+def upload_content(
+    user_id: str,
+    company_id: str,
+    path_to_content: str,
+    content_name: str,
+    mime_type: str,
+    scope_id: str | None = None,
+    chat_id: str | None = None,
+    skip_ingestion: bool = False,
+):
+    """
+    Uploads content to the knowledge base.
+
+    Args:
+        user_id (str): The user ID.
+        company_id (str): The company ID.
+        path_to_content (str): The path to the content to upload.
+        content_name (str): The name of the content.
+        mime_type (str): The MIME type of the content.
+        scope_id (str | None): The scope ID. Defaults to None.
+        chat_id (str | None): The chat ID. Defaults to None.
+        skip_ingestion (bool): Whether to skip ingestion. Defaults to False.
+
+    Returns:
+        Content: The uploaded content.
+    """
+
+    try:
+        return _trigger_upload_content(
+            user_id=user_id,
+            company_id=company_id,
+            path_to_content=path_to_content,
+            content_name=content_name,
+            mime_type=mime_type,
+            scope_id=scope_id,
+            chat_id=chat_id,
+            skip_ingestion=skip_ingestion,
+        )
+    except Exception as e:
+        logger.error(f"Error while uploading content: {e}")
+        raise e
+
+
+def _trigger_upload_content(
```

Comment:
Same comment as above, we can't have `_` prefix since it's not a private method but a utility function as such, the `_` designation wouldn't matter as it's ultimately a private method.